sktime/forecasting/base/_base.py

```diff
@@ -2305,6 +2347,27 @@ def _predict_moving_cutoff(
                 )
         return _format_moving_cutoff_predictions(y_preds, cutoffs)
 
+    def _get_varnames(self, default=None, legacy_interface=True):
+        """Return variable column for DataFrame-like returns.
+
+        Developer note: currently a helper for predict_interval, predict_quantiles,
+        valid only in the univariate case. Can be extended later.
+        """
+        if legacy_interface:
+            var_name = default
+        else:
+            y = self._y
+            if isinstance(y, pd.Series):
+                var_name = self._y.name
+            elif isinstance(y, pd.DataFrame):
+                return y.columns
```

Comment:
Personally, I prefer if functions have only one return or if each if-else clause has its own return.  Would it be possible to do something like that:

``python`
if isinstance(y, pd.Series):
  return [y.name] if y.name is not None else [0] # <- Probably if y.name is not None could also be part of the outer if case
elif isinstance(y, pd.DataFrame):
  return y.columns
else:
 return [0]

```